pkg/services/sqlstore/user.go

```diff
@@ -319,10 +319,15 @@ func SearchUsers(query *m.SearchUsersQuery) error {
 
 func DeleteUser(cmd *m.DeleteUserCommand) error {
 	return inTransaction(func(sess *xorm.Session) error {
-		deletes := []string{
-			"DELETE FROM star WHERE user_id = ?",
-			"DELETE FROM user WHERE id = ?",
-		}
+    var deletes [2]string
+
+    if (sess.Engine.DriverName() == "postgres") {
```

Comment:
this is not required, in other places I use the dialect abstraction to quote strings. 

https://github.com/grafana/grafana/blob/master/pkg/services/sqlstore/user.go#L280